pkg/controllers/node/termination/terminator/eviction.go

```diff
@@ -178,7 +178,7 @@ func (q *Queue) Evict(ctx context.Context, key QueueKey) bool {
 			}}, fmt.Errorf("evicting pod %s/%s violates a PDB", key.Namespace, key.Name)))
 			return false
 		}
-		logging.FromContext(ctx).Errorf("evicting pod, %s", err)
+		log.FromContext(ctx).Error(err, "Eviction error")
```

Comment:
same here on decapitalizing
```suggestion
		log.FromContext(ctx).Error(err, "eviction error")
```